pkg/remote/google/provider.go

```diff
@@ -55,10 +53,8 @@ func (p *GCPTerraformProvider) Version() string {
 	return p.version
 }
 
-func (p *GCPTerraformProvider) GetConfig() config.GCPTerraformConfig {
+func (p *GCPTerraformProvider) SetConfig(scopes []string) config.GCPTerraformConfig {
 	return config.GCPTerraformConfig{
-		Project: os.Getenv("CLOUDSDK_CORE_PROJECT"),
-		Region:  os.Getenv("CLOUDSDK_COMPUTE_REGION"),
-		Zone:    os.Getenv("CLOUDSDK_COMPUTE_ZONE"),
```

Comment:
Why do you removed Region and Zone parameters ?

Reply:
Because they are not needed since we use scopes.